src/formatDate.js

```diff
@@ -8,7 +8,50 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
```

Comment:
It's assumed that the separator is always the last element in the `fromFormat` array. While this works for the given examples, it might be safer to explicitly search for the separator in the `fromFormat` array to avoid potential issues if the array structure changes.